### DISCUSSION TITLE
tinymce changed its license to MIT.  Does mosaico want to change too?

### DISCUSSION THREAD

**Discussion by captproton** [voidlabs/mosaico#665] [Ideas]
I noticed that tinymce seems to be a mosaic dependency.  The current license is copyleft.  Now that tinymce is no longer copyleft, do you want to have a more permissive license?

**Comment by bago** [comment]
Mosaico licensing does not depend on Tinymce licensing as they are not "coupled".

That said, Tinymce 6.0 (the one being licensed as MIT) does not work with Mosaico even after the latest changes made recently, as you can read in #644.

Anyway we could have licensed Mosaico as MIT even before, but we decided for GPL because we also sell a non-GPL version of mosaico and this version is the one that pays for the development. Moving to a MIT license would need a new business model and we don't have one, at this time.
